Makefile

```diff
@@ -21,7 +21,7 @@ clean:
 	./gradlew clean
 
 build/distributions/tiered-storage-for-apache-kafka-$(VERSION).tgz:
-	./gradlew check releaseTarGz
+	./gradlew build releaseTarGz
```

Comment:
Probably running at least unit tests would still make sense for building the Docker image

Reply:
Build depends on check, check depends on both types of tests.